allennlp_models/common/model_card.py

```diff
@@ -42,6 +58,60 @@ def __str__(self):
 class ModelDetails(ModelCardInfo):
     """
     This provides the basic information about the model.
+
+    # Parameters:
+
+    description: `str`
+        A high-level overview of the model.
+        Eg. The model implements a reading comprehension model patterned
+            after the proposed model in [Devlin et al, 2018]
+            (https://api.semanticscholar.org/CorpusID:52967399), with improvements
+            borrowed from the SQuAD model in the transformers project.
+            It predicts start tokens and end tokens with a linear layer on top of
+            word piece embeddings.
+
+    developed_by: `str`
+        Person/organization that developed the model. This can be used by all
+        stakeholders to infer details pertaining to model development and
+        potential conflicts of interest.
+
+    contributed_by: `str`
+        Person that contributed the model to the repository.
+
+    date: `str`
+        The date on which the model was contributed. This is useful for all
+        stakeholders to become further informed on what techniques and
+        data sources were likely to be available during model development.
+        Format example: 2020-09-23
+
+    version: `str`
+        The version of the model, and how it differs from previous versions.
+        This is useful for all stakeholders to track whether the model is the
+        latest version, associate known bugs to the correct model versions,
+        and aid in model comparisons.
```

Comment:
I would just expect a number here, or a number with dots. It might even be nice if these were comparable, so you can find out which one is more recent programmatically. I can't square that with writing down "how it differs from previous versions". How do you intend for this to be used?

Reply:
I think the intention for "how it differs from previous versions" is something like a changelog. Eg. "v1.2; we retrained the model using dataset-2.0, which fixes bug x observed in v1.1".